dds/DCPS/TopicImpl.cpp

```diff
@@ -146,7 +146,8 @@ TopicImpl::enable()
                                              topic_name_.c_str(),
                                              type_name_.c_str(),
                                              qos_,
-                                             type_support_->has_dcps_key());
+                                             type_support_ ? type_support_->has_dcps_key() : false,
```

Comment:
Can type_support_ be null?  Should we be checking this in an earlier step and bailing out?

Reply:
I believe it can be null for recorder/replayer.